src/update-feeds-loops/update-feeds-loops.ts

```diff
@@ -171,7 +171,7 @@ export const runUpdateFeeds = async (providerName: ProviderName, chain: Chain, c
         if (goFirstBatch.data === null) return;
         const { batch: firstBatch, activeDataFeedCount, blockNumber: firstBatchBlockNumber } = goFirstBatch.data;
         if (activeDataFeedCount === 0) {
-          logger.warn(`No active data feeds found.`);
+          logger.info(`No active data feeds found.`);
```

Comment:
This is too noisy in Grafana and is an expected (or better, possible) scenario so I downgraded this to INFO. I still think it's a valuable information so I don't want to decrease to DEBUG.